docs/general/administration/hardware-selection.md

```diff
@@ -68,13 +74,11 @@ The CPU will be responsible for many tasks, notably:
 - Transcoding Audio
 - Decoding video in unsupported codecs
 
-Audio codecs are very lightweight and most CPUs should be able to handle them without issues. Video in unsupported codecs are usually older formats that are easier to decode. Assuming hardware acceleration is properly configured, any modern CPU with 4 threads should be able to handle the workload.
-
-:::caution
+Audio codecs are very lightweight and most CPUs should be able to handle them without issues (Ryzen 5950X FLAC -> AAC transcoding runs at ~110x real time). Please note that they will only use one core.
 
-Newer codecs can also require software decoding if there are no available hardware decoders. High bitrate HEVC content such as commonly found in 4k HDR can be very demanding even on modern CPUs. Please considering adding a modern GPU with 10bit HEVC decoding support. HDR to SDR tone-mapping is also currently not available without hardware acceleration.
+Video in unsupported codecs are usually older formats that are easier to decode. Assuming hardware acceleration is properly configured, any modern CPU with 4 threads should be able to handle the workload.
 
-:::
+However, newer codecs can also require software decoding if there are no available hardware decoders. HEVC, VP9 and AV1 can be **VERY** demanding even on modern CPUs if hardware acceleration isn't available. This is why a hardware platform capable of HEVC 10bit hardware decoding is strongly recommended.
```

Comment:
We have some software optimization for AV1 specifically. AV1 has a better optimized dav1d decoder, and we are enabling it in our custom ffmpeg. This decoder outperforms other software modern codecs like HEVC and VP9, and can provide good enough playback for most videos on modern CPUs. An Apple M1 can software decode up to 8K at 60fps (using all cores), although actual transcoding performance may be lower as the CPU has to be used for other tasks. As long as your CPU isn't too underpowered, you can decode 4K AV1 with this decoder without affecting playback too much, even an i3-7100 will do the trick (at the cost of 100% CPU load) if you are using hardware encoding with this software decoder.

This is helpful for users with older hardware that does not support AV1 but is still capable of decoding other codecs. We can let our users know that they don't have to panic if their hardware isn't new enough to support AV1 decoding because this codec has a better decoder and will have less impact on performance. Of course, it's still hard to beat hardware decoders and it will be less power-efficient. However, users can choose between buying a new GPU or saving money and using current hardware, especially when the GPU is integrated.

Reply:
4K AV1 pinning i3-7100 is still quite heavy considering what type of devices we usually see users running JF on so I am against removing AV1 from the list. 

I would add something like "The AV1 software decoder is more efficient than HEVC and VP9, and AV1 media isn't common yet, so it shouldn't be an issue to not have AV1 decoding for now."

Reply:
That decoder (dav1d) is not enabled in all ffmpeg builds and is not the default one comes with ffmpeg, so you may need to mention that as well. Our jellyfin-ffmpeg is built with that decoder.